### PR TITLE
[centos] Use latest version of Glusterfs

### PR DESCRIPTION
**Pull request by black-dragon74**
Signed-off-by: black-dragon74 <nickk.2974@gmail.com>

### CODE REVIEW

File: CentOS/Dockerfile [hunk 1]
```diff
@@ -34,6 +34,7 @@ yum --setopt=tsflags=nodocs -y install rsync  && \
 yum --setopt=tsflags=nodocs -y install tar  && \
 yum --setopt=tsflags=nodocs -y install cronie  && \
 yum --setopt=tsflags=nodocs -y install xfsprogs  && \
+yum --setopt=tsflags=nodocs -y install http://mirror.centos.org/centos/7/os/x86_64/Packages/gperftools-libs-2.6.1-1.el7.x86_64.rpm  && \
```
Comment (nixpanic) [inline CentOS/Dockerfile:37]: installing from CentOS 7 on CentOS 9? there should be a gperftools package for CentOS 9 somewhere too

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: CentOS/Dockerfile (hunk 1 above)
Comment (black-dragon74) [reply CentOS/Dockerfile:37]: Actually there isn't. It's not there even for CentOS 8. As this is related to tcmalloc, using older version is fine as we don't have any changes that might break stuff in the core gluster project.

Comment (nixpanic) [reply CentOS/Dockerfile:37]: what requires this then? do you have some installation or runtime error about it?

Comment (black-dragon74) [reply CentOS/Dockerfile:37]: Give me some time, I'll get it for you. It's tcmalloc IIRC, will confirm though.

Comment (nixpanic) [reply CentOS/Dockerfile:37]: Packages that are missing should get included in the CentOS Storage SIG repositories, if they are required for some Gluster components, that is. Installing them from somewhere else than standard repositories is always wrong.

Comment (black-dragon74) [reply CentOS/Dockerfile:37]:
```
nothing provides libtcmalloc_minimal.so.4()(64bit) needed by glusterfs-server-10.2-1.el9s.x86_64
```

It is `libtcmalloc`

Regards

Comment (black-dragon74) [reply CentOS/Dockerfile:37]: > Packages that are missing should get included in the CentOS Storage SIG repositories

Could you please look into this? `gperftools-libs` needs to be added. Would be also nice to have it available on CentOS 8.

Regards

Comment (nixpanic) [reply CentOS/Dockerfile:37]: the package is tagged and should land on http://mirror.stream.centos.org/SIGs/9-stream/storage/x86_64/gluster-10/Packages/g/ soon now too.

Comment (black-dragon74) [reply CentOS/Dockerfile]: We need `libunwind` too as `gperftools-libs` require it. Can you please tag it? Ref [here](https://centos.pkgs.org/8/epel-x86_64/libunwind-1.3.1-3.el8.x86_64.rpm.html).

Regards

Comment (nixpanic) [reply CentOS/Dockerfile]: The package comes from the CentOS Community Build System, at https://cbs.centos.org/koji/buildinfo?buildID=34022

This has now been added to the storage9s-gluster-10-release repository, so it should land on the mirrors soon.

**Review by nixpanic — APPROVED** [review]

**Review by humblec — APPROVED** [review]

**Pull request merged**